markovic/YahooNewsFeed/app/build.gradle

```diff
@@ -23,4 +23,6 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:22.2.0'
     compile 'com.android.support:support-v4:22.2.0'
+    compile 'com.android.support:recyclerview-v7:21.0.0'
```

Comment:
I think you can use  "compile 'com.android.support:design:22.2.0'" instead of 'com.android.support:recyclerview-v7:21.0.0' and 'com.android.support:appcompat-v7:22.2.0'. Please try.